app/experimenter/experiments/tests/serializers/test_timeline_population.py

```diff
@@ -189,14 +189,14 @@ def test_serializer_outputs_expected_schema_rollout(self):
     def test_serializer_saves_rollout_playbook_field(self):
         experiment = ExperimentFactory.create(
             type=ExperimentConstants.TYPE_ROLLOUT,
-            rollout_playbook="Low Risk Schedule",
+            rollout_playbook="low_risk",
```

Comment:
Use the value constant here and in the other two:

https://github.com/mozilla/experimenter/blob/master/app/experimenter/experiments/constants.py#L25